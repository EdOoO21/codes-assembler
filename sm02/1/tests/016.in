-40 -16 -417 -147 -536 -86 -989 -162 749 346 456 -470 -513 300 -545 -980 -393 -382 668 -314 365 -710 949 522 233 -365 -955 616 -549 958 234 -481 -959 829 -685 654 241 366 292 -944 -49 -65 226 283 440 -394
