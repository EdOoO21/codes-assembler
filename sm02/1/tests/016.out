-394
440
283
226
-65
-49
-944
292
366
241
654
-685
829
-959
-481
234
958
-549
616
-955
-365
233
522
949
-710
365
-314
668
-382
-393
-980
-545
300
-513
-470
456
346
749
-162
-989
-86
-536
-147
-417
-16
-40
